ci/deploy.sh

```diff
@@ -2,8 +2,11 @@
 
 set -e
 
+# Remove this after testing that it has the correct value.
+echo $GITHUB_REF
```

Comment:
From the [documentation,](https://docs.github.com/en/free-pro-team@latest/actions/reference/environment-variables#default-environment-variables) I think this will have the value:
```
refs/tags/v0.10.1
```

Reply:
I'll test it in a fork

Reply:
A simple solution in that case would be to:
```
$ echo $GITHUB_REF
refs/tags/v0.10.1
$ echo ${GITHUB_REF/refs\/tags\//}
v0.10.1
```

Reply:
This support page also seems to suggest that its `ref/tags/$TAGNAME`: https://github.community/t/how-to-get-just-the-tag-name/16241